bin/server

```diff
@@ -17,30 +17,21 @@ const executor = new ExecutorPlugin(executorConfig[executorConfig.plugin]);
 // Setup Login
 const loginConfig = config.get('login');
 
-loginConfig.datastore = datastore;
-
 // Setup httpd
 const httpdConfig = config.get('httpd');
```

Comment:
You can just set this to `const httpd = config.get('httpd');` and replace line 27 with `httpd,`